src/Symfony/Component/Security/Http/Firewall/SwitchUserListener.php

```diff
@@ -126,18 +126,18 @@ private function attemptSwitchUser(Request $request, $username)
             throw new \LogicException(sprintf('You are already switched to "%s" user.', $token->getUsername()));
         }
 
-        if (false === $this->accessDecisionManager->decide($token, array($this->role))) {
+        $user = $this->provider->loadUserByUsername($username);
+
+        if (false === $this->accessDecisionManager->decide($token, array($this->role), $user)) {
             $exception = new AccessDeniedException();
             $exception->setAttributes($this->role);
-
```

Comment:
this change should be reverted

Reply:
Why should this change be reverted?

Reply:
I'm sorry, I don't follow.  Are you saying that the $user param does not need to be passed to the accessDecisionManager->decide() call as the 3rd parameter?

Reply:
Oh that's a misunderstanding then. I was just referring to the blank line which should not have been removed.